### ISSUE TITLE
Train a two-linear-layer model on PeakNet20k to mimic the PCA analysis

### ISSUE THREAD

**Issue by carbonscott** [carbonscott/exp-peaknet#18]
Related to `proj-maxie` but it is here just to facilitate experiment management.

**Comment by carbonscott** [comment]
hmm... it's over 1B parameters... too inefficient to be really worth it

**Comment by carbonscott** [comment]
Trying patch based approach with patch size 60x60.  Change the underlying network to be a transformer.

**Comment by carbonscott** [comment]
s3df-issue18-3.0: loss = 0.21
